core/storage/suggestion/gae_models.py

```diff
@@ -3445,9 +3445,8 @@ class TranslationCoordinatorsModel(base_models.BaseModel):
     # The number of coordinators of this language. This property is added to
     # enable the sorting of datastore query results. It is equal to the
     # length of the coordinator_ids field.
-    # TODO(#18762): Validation for TranslationCoordinatorsModel:
-    # Add a validate method in domain layer to match the coordinators_count
-    # with length of coordinator_ids.
+    # TODO(#18762): Add a validate method in domain layer to match the
+    # coordinators_count with length of coordinator_ids.
```

Comment:
...to verify that the coordinators_count equals the length of coordinator_ids.

Reply:
Updated.